protocol/messenger_handler.go

```diff
@@ -1604,3 +1612,10 @@ func (m *Messenger) updateUnviewedCounts(chat *Chat, mentioned bool) {
 		chat.UnviewedMentionsCount++
 	}
 }
+
+func (m *Messenger) decreamentUnviewedCounts(chat *Chat, mentioned bool) {
```

Comment:
Please also rename `updateUnviewedCounts` function to `increamentUnviewedCounts` (So that they both look similar)

Reply:
cool